utils/server_handler.py

```diff
@@ -0,0 +1,125 @@
+import http.client
+import subprocess
+import os
+
+DEFAULT_SERVER_LOCATION = "C:/Users/Matt/Dev/CS52-Autograder/serve-sml"
+DEFAULT_FILE_ORDER = ["buffer", "util", "http", "server", "assert", "grading_lib", "main"]
+
+class ServerHandler:
+
+    def __init__(self, server_location:str=None, files:list=None):
+        """Make the Server object
+
+        Args:
+        Server_location:  Where the sml files are located
+        files:  the files (and order) that they should be run in
+        """
+
+        if server_location is not None:
```

Comment:
alternatively this block could read `self.server_location = server_location or DEFAULT_SERVER_LOCATION`.
similar below.